packages/synapse-react-client/src/components/SynapseHomepageV2/SynapseSearchChips.tsx

```diff
@@ -0,0 +1,112 @@
+import {
+  SearchQuery,
+  KeyValue,
+  SearchFieldName,
+  SearchFacetSort,
+} from '@sage-bionetworks/synapse-types'
+
+const searchValues = [
+  "Alzheimer's Disease",
+  'Neurofibromatosis mouse models',
+  'Cancer',
+  'UK Biobank',
+  'Single-cell analysis of breast cancer',
+  'ROSMAP Religious Orders Study and Memory and Aging Project',
+  'Retinal regeneration studies',
+  'Machine Learning datasets',
+]
+
+const projectKeyValue: KeyValue = {
+  key: 'node_type',
+  value: 'project',
+  not: false,
+}
+
+export type SynapseSearchChipsProps = {
+  gotoPlace: (href: string) => void
+}
+
+export const SynapseSearchChips: React.FunctionComponent<
+  SynapseSearchChipsProps
+> = ({ gotoPlace }) => {
+  return (
+    <>
+      {searchValues.map(value => {
+        return (
+          <Chip
+            key={value}
+            label={
+              <Typography
+                variant="body1"
+                sx={{
+                  fontSize: '18px',
+                  fontWeight: 400,
+                  p: '10px',
+                }}
+              >
+                {value}
+              </Typography>
+            }
+            onClick={() => {
+              const searchQuery: SearchQuery = {
+                queryTerm: [value],
+                booleanQuery: value == 'Cancer' ? [projectKeyValue] : undefined,
+                // match existing facet options in SWC search
+                facetOptions: [
+                  {
+                    name: SearchFieldName.ENTITY_TYPE,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                  {
+                    name: SearchFieldName.CONSORTIUM,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                  {
+                    name: SearchFieldName.MODIFIED_ON,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                  {
+                    name: SearchFieldName.MODIFIED_BY,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                  {
+                    name: SearchFieldName.CREATED_ON,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                  {
+                    name: SearchFieldName.TISSUE,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                  {
+                    name: SearchFieldName.CREATED_BY,
+                    maxResultCount: 300,
+                    sortType: SearchFacetSort.COUNT,
+                  },
+                ],
+                start: 0,
+                size: 30,
+              }
+              gotoPlace(`/Search:${JSON.stringify(searchQuery)}`)
```

Comment:
Verified integration with local SWC (which is how I discovered that I needed to set the facet options)